### ISSUE TITLE
Change default dataloader_num_workers in trainers and testers

### ISSUE THREAD

**Issue by KevinMusgrave** [KevinMusgrave/pytorch-metric-learning#361]
32 is usually too high
